src/main/java/seedu/anilist/model/anime/Anime.java

```diff
@@ -19,21 +19,37 @@
     private final Name name;
 
     // Data fields
+    private final Episode episode;
     private final Set<Tag> tags = new HashSet<>();
 
     /**
-     * Every field must be present and not null.
+     * Every field must be present and not null except Episode which defaults to 0.
      */
     public Anime(Name name, Set<Tag> tags) {
         requireAllNonNull(name, tags);
         this.name = name;
+        this.episode = new Episode("0");
+        this.tags.addAll(tags);
+    }
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Anime(Name name, Episode episode, Set<Tag> tags) {
+        requireAllNonNull(name, tags, episode);
+        this.name = name;
+        this.episode = episode;
         this.tags.addAll(tags);
     }
```

Comment:
Has this been linked to the addAnimeCommand yet? 
i.e. you can specify the episode while adding an anime.

Reply:
I don't believe so, I feel like that should be on another branch as the current branch is getting too large. Maybe I'll add this issue.